packages/design-system-react-native/src/components/AvatarAccount/AvatarAccount.figma.tsx

```diff
@@ -0,0 +1,44 @@
+import codeConnect from '@figma/code-connect';
```

Comment:
This is the main Figma component code connect file. https://www.figma.com/code-connect-docs/react/
I considered trying to push it to include BadgeWrapper for the badges but I think we can update it when we add badge support and can keep this PR reasonably size